src/pages/Lend/Filter/FilterRangeSlider.vue

```diff
@@ -100,16 +98,65 @@ export default {
 			default: null,
 			required: true
 		},
+		range: {
+			type: Object,
+			default: null,
+			required: true
+		},
+		refine: {
+			type: Function,
+			default: null,
+			required: true
+		},
+	},
+	computed: {
+		// set label for slider
+		rangeLabel() {
+			let value = this.maximumLabel;
+
+			if (this.selectedRangeValue === this.maximum) {
+				value = this.maximumLabel;
+
+			// minimum value
+			} else if (this.selectedRangeValue === this.minimum) {
+				if (this.minimumValueLabel === false) {
+					value = this.minimumLabel;
+				} else {
+					value = `${this.selectedRangeValue}${this.minimumLabel}`;
+				}
+
+			// values in between min/max value
+			} else {
+				value = `${this.selectedRangeValue}${this.inbetweenLabel}`;
+			}
+
+			return value;
+		},
+	},
+	watch: {
+		// upon deletion of refinement (filter menu closed) update range: label, value and style (liner gradient)
+		currentRefinement(data) {
+			if (this.filterMenuOpen === false) {
+				if (data.min === null && data.max === null) {
```

Comment:
Why/when does minimum value for a max only slider change from -1 to null?

Reply:
@ksekhri The max/min value(s) are set to null when the refinement is removed from the SelectedRefinements.